contribs/gmf/less/desktop.less

```diff
@@ -184,6 +185,7 @@ main {
     float: right;
     height: 100%;
     position: relative;
+    z-index: 2;
```

Comment:
Nope, I don't want to use this variable here since it doesn't mean anything in this context. IMO, using variables for indexes is useful for elements with absolute positioning and that are in the same container.